app/mesh/1.0.x/installation/amazonlinux.md

```diff
@@ -44,12 +45,24 @@ $ tar xvzf kong-mesh-{{page.kong_latest.version}}*.tar.gz
 {% endnavtab %}
 {% endnavtabs %}
 
-### 2. Follow Kuma Instructions
+### 2. Configure The License
```

Comment:
Where does the license come from? Is it granted by sales, self-serve in Bintray, or something else?

Reply:
Provided by our sales/support team.